tests/test_on_data/test_metadata/test_scanimage_metadata.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+
+import numpy as np
+
+from nwb_conversion_tools.datainterfaces.ophys.scanimage.scanimageimaginginterface import ScanImageImagingInterface
+
+from ..setup_paths import OPHYS_DATA_PATH
+
+
+def test_scanimage_metadata():
+    scan_image_tiff_filepath = OPHYS_DATA_PATH / "imaging_datasets" / "Tif" / "sample_scanimage.tiff"
+
+    scaniamge_data_interface = ScanImageImagingInterface(scan_image_tiff_filepath)
+    assert scaniamge_data_interface.get_metadata() == {
+        "Ophys": {
+            "Device": [{"name": "Microscope"}],
+            "ImagingPlane": [
+                {
+                    "name": "ImagingPlane",
+                    "description": "no description",
+                    "excitation_lambda": np.nan,
+                    "indicator": "unknown",
+                    "location": "unknown",
+                    "optical_channel": [
+                        {"name": "channel_0", "emission_lambda": np.nan, "description": "no description"}
+                    ],
+                    "imaging_rate": 3.90625,
```

Comment:
Is this really the sampling rate? I thought usually they were around 30 Hz, or is it just because this is a test file?

Reply:
This is really the sampling rate for real experimental data. You are correct that it is unusually low.